test-app/ember-cli-build.js

```diff
@@ -15,11 +19,11 @@ module.exports = function (defaults) {
     behave. You most likely want to be modifying `./index.js` or app's build file
   */
 
-  if (app.env === 'test') {
-    app.import('node_modules/ember-source/dist/ember-template-compiler.js', {
-      type: 'test',
-    });
-  }
+  // if (app.env === 'test') {
```

Comment:
why comment this out and leave it?

Reply:
I didn't know what it does. So I commented it to see if it has an affect. Do you know what it does?